app.py

```
@@ -3,7 +3,7 @@
 import pickle
 
 DB_USER = "admin"
-DB_PASS = "SuperSecret1337!"
+DB_PASS = "superSecret1337!"
```

Ruff (0.14.10)

Possible hardcoded password assigned to: "DB_PASS"

(S105)